src/ert/enkf_main.py

```diff
@@ -148,6 +150,26 @@ def _seed_sequence(seed: Optional[int]) -> int:
     return int_seed
 
 
+def load_prior(
+    ensemble: Ensemble, active_realizations: Iterable[int], design_matrix: DesignMatrix
+) -> None:
+    assert design_matrix.parameter_configuration is not None
+    assert (
+        design_matrix.design_matrix_df is not None
```

Comment:
Perhaps better to do in a different PR, but could we instead just call it `df` so that we get `design_matrix.df`?

Reply:
we usually try to avoid using abbreviations but this one might be ok :+1:

Reply:
It's a common naming convention in pandas.

Reply:
I'm aware of df in pandas but not sure if all are. Anyway this can be changed in another pr.